beacon_node/eth2_libp2p/src/peer_manager/client.rs

```diff
@@ -30,6 +30,8 @@ pub enum ClientKind {
     Teku,
     /// A Prysm node.
     Prysm,
+    /// A loadstar node.
```

Comment:
One more :joy:

Reply:
Damn. You're on it! Eagle eye Michael. 

Thanks :blush: